crane/app_util.py

```diff
@@ -107,9 +114,16 @@ def wrapper(image_id, *args, **kwargs):
             if not repo_tuple.protected:
                 found_match = True
                 break
-            elif cert and cert.check_path(repo_tuple.url_path):
-                found_match = True
-                break
+            elif repo_tuple.protected:
+                value = repo_tuple.url_path
+                logger.info('path before check for images %s' % value)
+                if 'webassets/docker' in value:
```

Comment:
Using `in` here (and in the other places this code is repeated) looks wrong. Should it be `.startswith()`?

Reply:
value contains 'webassets/docker' in between so I have used `in` here

Reply:
But the next line strips off a string the length of 'webassets/docker', so if it wasn't at the start won't it remove the wrong characters?

Reply:
value generated is ''/webassets/docker/content/dist/rhel/server/7/7Server/x86_64/containers/registry/'

Let me make this startswith('/webassets/docker')